packages/editor/src/core/extensions/custom-image/components/image-block.tsx

```
@@ -59,12 +59,12 @@ export const CustomImageBlock: React.FC<CustomImageBlockProps> = (props) => {
     src: remoteImageSrc,
     setEditorContainer,
   } = props;
-  const { width, height, aspectRatio } = node.attrs;
+  const { src: remoteImageSrc, width: nodeWidth, height: nodeHeight, aspectRatio: nodeAspectRatio } = node.attrs;
```

Biome

[error] Shouldn't redeclare 'remoteImageSrc'. Consider to delete it or rename it.

'remoteImageSrc' is defined here:


(lint/suspicious/noRedeclare)